src/broker/content-cache.c

```diff
@@ -312,6 +312,14 @@ static int cache_entry_fill (struct content_cache *cache,
                                   e->data,
                                   e->len,
                                   "load");
+
+        /* Push entry onto front of LRU upon transition from invalid to valid,
+         * but only if entry is not dirty.
+         */
+        if (!e->dirty) {
+            cache_lru_push (cache, e);
+            e->lastused = flux_reactor_now (cache->reactor);
```

Comment:
given how the code ended up, could the `lastused` update be moved into lru push?

Reply:
Will do.